platform/src/main/java/org/stellar/anchor/platform/config/PropertyClientsConfig.java

```diff
@@ -105,7 +105,7 @@ public void validateNonCustodialClient(ClientConfig clientConfig, Errors errors)
     if (!isEmpty(clientConfig.getSigningKey())) {
       try {
         String clientSigningKey =
-            Sep10Helper.fetchSigningKeyFromClientDomain(clientConfig.getDomain());
+            Sep10Helper.fetchSigningKeyFromClientDomain(clientConfig.getDomain(), false);
```

Comment:
Question: why is this hardcoded to false? I thought it depended on whether or not we were using Testnet.

Reply:
I didn't want to complicate the code for the only use case (where anchor uses optional key verification configuration for non-custodial AND the client server uses http)
I don't think there should be a wallet server that uses http anywhere. This flag was more of a hack to enable wallet server.